Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -94,14 +94,18 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
 
     let backing = project.map { $0.personalization.backing }.skipNil()
     self.showPledgeOverTimeUI = project.signal
-      .map { ($0.isPledgeOverTimeAllowed ?? false) && featurePledgeOverTimeEnabled()
-      }
+      .map { _ in featurePledgeOverTimeEnabled() }
 
     self.pledgeOverTimeConfigData = self.showPledgeOverTimeUI
-      .map { showPledgeOverTimeUI -> PledgePaymentPlansAndSelectionData? in
+      .combineLatest(with: project)
+      .map { showPledgeOverTimeUI, project -> PledgePaymentPlansAndSelectionData? in
         guard showPledgeOverTimeUI else { return nil }
```

Comment:
nit: this guard statement could be replaced with a `.filter(showPledgeOverTimeUI)`. Then this output wouldn't need to return an optional `PledgePaymentPlansAndSelectionData`